spec/system/consumer/account_spec.rb

```diff
@@ -73,8 +73,8 @@
                                   href: "#{distributor2.permalink}/shop", count: 1)
         expect(page).not_to have_content distributor_without_orders.name
 
-        expect(page).to have_content distributor1.name + "\n" + "Balance due"
-        expect(page).to have_content distributor_credit.name + "\nCredit"
+        expect(page).to have_content distributor1.name + " Balance due"
```

Comment:
I do get those warnings locally as well. Worth fixing. The auto-correction does this:
```diff
diff --git a/spec/system/consumer/account_spec.rb b/spec/system/consumer/account_spec.rb
index c58e540db..f108006c6 100644
--- a/spec/system/consumer/account_spec.rb
+++ b/spec/system/consumer/account_spec.rb
@@ -73,15 +73,15 @@ describe '
                                   href: "#{distributor2.permalink}/shop", count: 1)
         expect(page).not_to have_content distributor_without_orders.name
 
-        expect(page).to have_content distributor1.name + " Balance due"
-        expect(page).to have_content distributor_credit.name + " Credit"
+        expect(page).to have_content "#{distributor1.name} Balance due"
+        expect(page).to have_content "#{distributor_credit.name} Credit"
 
         # It reveals table of orders for distributors when clicked
         expand_active_table_node distributor1.name
-        expect(page).to have_link "Order " + d1o1.number, href: "/orders/#{d1o1.number}"
+        expect(page).to have_link "Order #{d1o1.number}", href: "/orders/#{d1o1.number}"
 
         expand_active_table_node distributor2.name
-        expect(page).not_to have_content "Order " + d1o1.number.to_s
+        expect(page).not_to have_content "Order #{d1o1.number}"
       end
 
       context "when there is at least one changeable order" do
```

Reply:
How do you get the _auto-correction_ @mkllnk ?

Reply:
There are two ways.
```sh
# safe auto-correct
bundle exec rubocop -a

# "unsafe" (needs review) auto-corrent
bundle exec rubocop -A
```

Reply:
This change needed `-A`.

Reply:
Thanks Maikel :+1: I'll need to double check why I could not find these locally.

Reply:
Alright, found them :+1: All good now, merging.